patches/net/minecraft/client/Minecraft.java.patch

```diff
@@ -177,14 +178,15 @@
          this.deltaTracker.updatePauseState(this.pause);
          this.deltaTracker.updateFrozenState(!this.isLevelRunningNormally());
          long l = Util.getNanos();
-@@ -1351,10 +_,12 @@
+@@ -1351,10 +_,13 @@
          this.window.setGuiScale((double)i);
          if (this.screen != null) {
              this.screen.resize(this, this.window.getGuiScaledWidth(), this.window.getGuiScaledHeight());
 +            net.neoforged.neoforge.client.ClientHooks.resizeGuiLayers(this, this.window.getGuiScaledWidth(), this.window.getGuiScaledHeight());
          }
  
          RenderTarget rendertarget = this.getMainRenderTarget();
++        if (rendertarget != null)
```

Comment:
What is this?

Reply:
The main render target may not exist by this point (e.g. due to a mod causing an error during loading) and the resize() call here would obscure the error logs. This is because we separated mod construction to allow access to configuration of the main render target.

Reply:
I don't see how that can possibly happen. If the event completes, the render target should be created.